src/main/webapp/js/selectors/index.js

```diff
@@ -21,3 +32,11 @@ export const getHiddenFields = createSelector(
 export const getFieldConfigById = (state, { entityType, fieldId }) => (
   state.fieldConfig[entityType][fieldId]
 );
+
+export const getEntityArrayByName = createSelector(
+  getEntityByName, entity => mapToArray(entity),
+);
+
+export const getEntityFieldValue = createSelector(
```

Comment:
you don't need to create a selector for simple value (e.g. string), also it will not work if the same selector is used in multiple components